code/modules/mob/living/silicon/ai/ai_mob.dm

```diff
@@ -1480,4 +1484,13 @@ GLOBAL_LIST_INIT(ai_verbs_default, list(
 	to_chat(src, "<span class='notice'>The tiny in built fan finally removes the tape!</span>")
 	ducttapecomponent.remove_tape(card, src)
 
+//Stores the location of the AI to the value of stored_locations associated with location_number.
+/mob/living/silicon/ai/proc/store_location(location_number)
+	if(!isturf(eyeobj.loc)) //i.e., inside a mech or other shenanigans
+		to_chat(src, "<span class='warning'>Failed to set location [location_number].</span>")
```

Comment:
This message could be a bit more direct, noting that they can't set it there, rather than it "failing"